cityofaustin/atd-vz-data
# Update years and percentage in VZV map overlays popover

## Associated issues
This is something that pertains to [#9891](https://github.com/cityofaustin/atd-data-tech/issues/9891) - there is no individual Issue for this because I considered it a very small change.  I've updated the dates and percentage to reflect the current copy from the [AGOL version](https://github.com/cityofaustin/atd-data-tech/issues/9891) of the updated HIN.

## Testing
**URL to test:** https://deploy-preview-1094--atd-vzv-staging.netlify.app/viewer/map

**Steps to test:**

1. Confirm that dates in the Overlay popup changed from 2013 to 2017 --> 2017-2021. 
2. Confirm percentage changed from 70% to 60%. 

---
#### Ship list
